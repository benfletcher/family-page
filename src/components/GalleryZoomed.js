import React from 'react';

const GalleryZoomed = props => (
  <div className="galleryZoomed">
    <div className="flexZoomedContainer">
      <div className="zoomedPhotoContainer">
        <div className="zoomedNavigation">
          <i
            className="fa fa-3x white fa-arrow-left zoomedNavigationLeft"
            onClick={props.goLeft}
            aria-hidden="true"
          />
          <i
            className="fa fa-3x white fa-arrow-right zoomedNavigationRight"
            onClick={props.goRight}
            aria-hidden="true"
          />
        </div>
        <i
          className="zoomedNavigationExit fa fa-times"
          aria-hidden="true"
          alt="close"
          onClick={props.zoom}
        />
        <div>
          <img className="zoomedPhoto" src={props.photoUrl} alt="large" />
        </div>
      </div>
    </div>
  </div>
);

GalleryZoomed.propTypes = {
  photoUrl: React.PropTypes.string.isRequired,
  zoom: React.PropTypes.func.isRequired,
  goLeft: React.PropTypes.func.isRequired,
  goRight: React.PropTypes.func.isRequired,
};

export default GalleryZoomed;
