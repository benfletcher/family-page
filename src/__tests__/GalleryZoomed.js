/* eslint-env browser, jest */
import React from 'react';
import renderer from 'react-test-renderer';
import GalleryZoomed from '../components/GalleryZoomed';

test('Header snapshot is the same', () => {
  const component = renderer.create(
    <GalleryZoomed />
  );
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
