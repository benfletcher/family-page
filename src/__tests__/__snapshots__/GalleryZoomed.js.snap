exports[`test Header snapshot is the same 1`] = `
<div
  className="galleryZoomed">
  <div
    className="flexZoomedContainer">
    <div
      className="zoomedPhotoContainer">
      <i
        alt="close"
        aria-hidden="true"
        className="xIcon fa fa-times"
        onClick={undefined} />
      <img
        alt="large"
        className="zoomedPhoto"
        src={undefined} />
    </div>
  </div>
  <div
    className="zoomedNav">
    <i
      aria-hidden="true"
      className="fa fa-3x white fa-arrow-left"
      onClick={undefined} />
    <i
      aria-hidden="true"
      className="fa fa-3x white fa-arrow-right"
      onClick={undefined} />
  </div>
</div>
`;
